src/ir/branch-utils.h

```diff
@@ -327,6 +327,12 @@ struct BranchAccumulator
     auto selfBranches = getUniqueTargets(curr);
     branches.insert(selfBranches.begin(), selfBranches.end());
   }
+
+  static NameSet get(Expression* tree) {
+    BranchAccumulator branches;
+    branches.walk(tree);
+    return branches.branches;
```

Comment:
I would find this naming clearer.
```suggestion
    BranchAccumulator accumulator;
    accumulator.walk(tree);
    return accumulator.branches;
```